app.js

```diff
@@ -141,7 +141,7 @@ app
     .use(router.allowedMethods());
 
 if (!module.parent) {
-    app.listen(process.env.INTERNAL_CONTRACT_HELPER_PORT || process.env.PORT || 3000);
+    app.listen(3000);
```

Comment:
`PORT` is still very useful for running locally

Reply:
Same question as in here: https://github.com/nearprotocol/near-fiddle-api/pull/8#discussion_r293169650